src/lib/resources.ts

```diff
@@ -270,4 +273,27 @@ export class Items extends OPResource {
 
         return ObjectSerializer.deserialize(data, "Array<ItemFile>");
     }
+
+
+    /**
+     * Get Item's OTP.
+     * itemQuery param can be an item's Title or ID.
+     *
+     * If there are more than one OTP field in an item
+     * it always returns the first/main one.
+     *
+     * @param {string} vaultId
+     * @param {string} itemQuery
+     * @returns {Promise<string>}
+     */
+    public async getOTP(vaultId: string, itemQuery: string): Promise<string> {
+        const item: FullItem = await this.get(vaultId, itemQuery);
+        const otp = item.extractOTP();
+
+        if (!otp) {
+            throw new Error(ErrorMessageFactory.noOTPFoundForItem(item.id));
+        }
```

Comment:
Would it make sense to make this check part of the `extractOTP` method?

Reply:
I'd say yes.
Users will understand right away that there is something wrong. And as all the methods in `resources` make the calls to Connect, and might throw errors, I think it's ok to throw it here also.

Reply:
Understood. Thank you for the clarification. Then we can keep this check only for the method in the `resources`.